sql/core/src/main/scala/org/apache/spark/sql/execution/joins/HashedRelation.scala

```diff
@@ -324,7 +324,9 @@ private[joins] object UnsafeHashedRelation {
  *
  * see http://java-performance.info/implementing-world-fastest-java-int-to-int-hash-map/
  */
-private[execution] final class LongToUnsafeRowMap(var mm: TaskMemoryManager, capacity: Int)
+private[execution] final class LongToUnsafeRowMap(
+    @transient val mm: TaskMemoryManager,
```

Comment:
It probably means the classes could be better separated into the "thing that needs the task manager" and the "data object" rather than conflate them, but it may happen to be OK here indeed.